app/javascript/packs/new_ad.js

```diff
@@ -0,0 +1,137 @@
+$(document).ready(function() {
```

Comment:
do not use multiple `$(document).ready(function()` keep everything under a single `$(document).ready(function()` wherever it is being used